sdk/storage/storage-blob/BreakingChanges.md

```diff
@@ -33,17 +33,17 @@
       includeUncommitedBlobs: true
     });
     ```
-- [Breaking] Fixed typo - `chanageLease` -> `changeLease`, a method on `LeaseClient`.
-- [Breaking] Rename `BlobClient.setTier()` to `BlobClient.setAccessTier()`.
-- [Breaking] `Models.StorageServiceProperties` is renamed into `Models.BlobServiceProperties`
+- Fixed typo - `chanageLease` -> `changeLease`, a method on `LeaseClient`.
+- Rename `BlobClient.setTier()` to `BlobClient.setAccessTier()`.
+- `Models.StorageServiceProperties` is renamed into `Models.BlobServiceProperties`
 
 ### 2019.09 Version 12.0.0-preview.3
 
 - `RawTokenCredential` is dropped. TokenCredential implementations can be found in the [@azure/identity](https://www.npmjs.com/package/@azure/identity) library for authentication.
 
 ### 2019.08 Version 12.0.0-preview.2
 
-- [Breaking] Aborter class is no longer exposed from the package. Use the package [@azure/abort-controller](https://www.npmjs.com/package/@azure/abort-controller) to pass an abort signal to any of the async operations.
```

Comment:
and here

Reply:
Same for file and queue

Reply:
It is redundant to have `[Breaking]` in BreakingChanges.md

Reply:
Ohhh.